adserver/templates/adserver/base.html

```diff
@@ -29,21 +29,24 @@
               <div class="dropdown-divider"></div>
             {% endif %}
 
+            <a class="dropdown-item" href="{% url 'account' %}">
+              <span class="fa fa-user fa-fw mr-2 text-muted" aria-hidden="true"></span>
+              <span>{% trans 'Account' %}</span>
```

Comment:
Feels a bit more explicit of what it is?

```suggestion
              <span>{% trans 'Account Settings' %}</span>
```